wolfssl/wolfcrypt/types.h

```diff
@@ -2050,7 +2050,7 @@ enum Max_ASN {
                             /* Maximum DER digest ASN header size */
                             /* Max X509 header length indicates the
                              * max length + 2 ('\n', '\0') */
-    MAX_X509_HEADER_SZ  = (37 + 2), /* Maximum PEM Header/Footer Size */
+    MAX_X509_HEADER_SZ  = (41 + 2), /* Maximum PEM Header/Footer Size */
```

Comment:
Oh but... `-----BEGIN SPHINCS_SMALL_LEVEL1 PRIVATE KEY-----` is 48. Should this be fixed for the correct max? Maybe even wrapped in a build time option for `#if defined(HAVE_SPHINCS)`  or `#if defined(HAVE_DILITHIUM)`?

Reply:
it turns out that `-----BEGIN SPHINCS_SMALL_LEVEL1 PRIVATE KEY-----` is the longest one at length 48.